### DISCUSSION TITLE
T3xtifyai / Octernship Submission no Pull request found for submission

### DISCUSSION THREAD

**Discussion by PK284** [community/community#51553] [GitHub Education]
In Textify project submission it is mentioned that the Students are expected to submit their work as either presentations/ video submissions/ live product demos, etc. Using the auto-generated Feedback Pull Request for review and submission Using GitHub Discussions to ask any relevant questions regarding the project
But I am unable to find any Auto-generated pull request in my repository.
Kindly help and do tell me which moderator I need to contact for my submission.
Kindly help me as my submission is due on the 1st of April.
![image](https://user-images.githubusercontent.com/72080733/229066000-61b52e56-79cf-4c2c-88c7-535d7642bbd8.png)
Link to the task page: https://github.com/T3xtifyai/introduction-PK284

**Comment by samus-aran** [comment]
Hi @PK284 👋 

We would recommend students working on Octernship projects to reach out to the project’s maintainers to ask for clarification on the proposed tasks or identify any issues they are experiencing. All project’s may have different preferred methods of communication but here are a few places to check where best to raise inquiry:

- Octernships Project’s README - This may include instructions on how and where to raise an inquiry.
- Octernships Project’s Discussions - Not all projects’ repos have this enabled. We encourage students to first search in case your question has already been answered.
- Octernships Project’s Issues - We encourage students to first search in case your question has already been answered. We also encourage students to follow instructions on the issue template(s) on how to create a suitable issue if provided.

Hope this helps 👍
